test/hotspot/jtreg/compiler/vectorization/runner/BasicFloatOpTest.java

```diff
@@ -35,7 +35,7 @@
  *                   -XX:+WhiteBoxAPI
```

Comment:
UseFMA is also needed for IR checks for other platforms like x86 avx and aarch64 asimd, right? Seems to me more reasonable to add a `-XX:+UseFMA` VM option when running this test. Then we won't need to add all these `applyIf = {"UseFMA", "true"}`.

Reply:
I suggest we don't touch other platforms unless I have to do so (e.g. in https://github.com/openjdk/jdk/pull/24947, I have to touch an existing test in another platform), because we need to test it, and it could introduce potential failures, so I won't risk for it.

For `-XX:+UseFMA`, I suggest we don't add it to the whole test, as there are other IR verifications, which does not depend on it.

How do you think about it?

Reply:
And, as you can see `"fma", "true", "avx", "true"`, it's doing the similar thing as riscv.

Reply:
> I suggest we don't touch other platforms unless I have to do so (e.g. in #24947, I have to touch an existing test in another platform), because we need to test it, and it could introduce potential failures, so I won't risk for it.

OK if you want to keep this change RISC-V specific.

> For `-XX:+UseFMA`, I suggest we don't add it to the whole test, as there are other IR verifications, which does not depend on it.

Make sense to me.

Reply:
Thank you!